styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -89,10 +88,10 @@
   private static final Logger LOG = LoggerFactory.getLogger(Scheduler.class);
 
   @VisibleForTesting
-  static final String GLOBAL_RESOURCE_ID = "GLOBAL_STYX_CLUSTER";
+  public static final String GLOBAL_RESOURCE_ID = "GLOBAL_STYX_CLUSTER";
```

Comment:
Might better just move this to `StateUtil`.

Reply:
I am not sure, it's also used in `Scheduler`